src/metabase/api/user.clj

```diff
@@ -64,7 +64,7 @@
            [400 (tru "Not able to modify the internal user")]))
 
 (defn- fetch-user [& query-criteria]
-  (apply t2/select-one (vec (cons User user/admin-or-self-visible-columns)) query-criteria))
+  (apply t2/select-one (vec (cons User user/admin-or-self-visible-columns)) :type "personal" query-criteria))
```

Comment:
Generally we use keywords in queries for these sorts of enum-like columns. Toucan2 will automatically convert them to strings because you've already defined a keyword transformation for this field on the User model.
```suggestion
  (apply t2/select-one (vec (cons User user/admin-or-self-visible-columns)) :type :personal query-criteria))
```

Reply:
Same applies to the instances below